### PR TITLE
added numeric keyboard support

### PR DESCRIPTION
**Pull request by parthmakadiya12**
Added Numeric Keyboard

pass a new prop
`keyinputmode:"numeric"`
to use the keyboard

### CODE REVIEW

**Comment by parthmakadiya12** [comment]
@afc163 @zombieJ please review if it can be merged

**Comment by zombieJ** [comment]
I'm ok with this prop. But maybe only describe in doc and not update the example. And may use `inputMode` directly instead.

cc @afc163 @yesmeck

**Comment by parthmakadiya12** [comment]
@zombieJ  ok i will change the name to `inputmode` (in small character and document it somewhere )
will let you know about this.

File: README.md [hunk 1]
```diff
@@ -249,6 +249,12 @@ http://react-component.github.io/calendar/examples/index.html
           <td></td>
           <td>specific the clear icon.</td>
         </tr>
+         <tr>
+          <td>inputMode</td>
+          <td>strimg</td>
+          <td>text</td>
+          <td>Change the keyboard in mobile device by passing "numeric"</td>
```
Comment (zombieJ) [inline README.md]:
```diff
- Change the keyboard in mobile device by passing "numeric"
+ Change the keyboard in mobile device
```

File: examples/antd-calendar.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ class Demo extends React.Component {
       defaultValue={this.props.defaultCalendarValue}
       showDateInput={state.showDateInput}
       disabledDate={disabledDate}
+      inputMode={"numeric"}
```
Comment (zombieJ) [inline examples/antd-calendar.js]: Needn't pass `inputMode` in example since we should keep example simple.

Comment (parthmakadiya12) [reply examples/antd-calendar.js]: Done

File: README.md (hunk 1 above)
Comment (parthmakadiya12) [reply README.md]: done

File: examples/antd-calendar.js [hunk 3]
```diff
@@ -174,6 +174,7 @@ class Demo extends React.Component {
                     tabIndex="-1"
                     className="ant-calendar-picker-input ant-input"
                     value={value && value.format(getFormat(state.showTime)) || ''}
+                    inputMode={"numeric"}
```
Comment (zombieJ) [inline examples/antd-calendar.js]: here ⤴️

File: rc-calendar-tests.tsx [hunk 4]
```diff
@@ -16,5 +16,6 @@ export default () => (
     onChange={action}
     disabledDate={(now: moment.Moment) => false}
     onSelect={action}
+    inputMode={"numeric"}
```
Comment (zombieJ) [inline rc-calendar-tests.tsx]: => `inputMode="numeric"`

File: examples/antd-calendar.js (hunk 3 above)
Comment (parthmakadiya12) [reply examples/antd-calendar.js]: done. I also remove it from other examples as well
Sorry about that

**Comment by parthmakadiya12** [comment]
@zombieJ  all changes you suggested are done
please check

**Comment by parthmakadiya12** [comment]
@zombieJ please merge it.
I had some merge conflict in last time you tried so now they are resolved.
Feel free to merge it.

Thanks

**Pull request merged**